bgstally/fleetcarrier.py

```diff
@@ -196,25 +211,54 @@ def get_items_plaintext(self, category: FleetCarrierItemType|None = None) -> str
         Returns:
             str: _description_
         """
-        result: str = ""
-        items, name_key, display_name_key, quantity_key = self._get_items(category)
-
-        if items is None: return ""
-        items = sorted(items, key=lambda x: x[name_key])
 
-        for item in items:
-            if category == FleetCarrierItemType.COMMODITIES_BUYING or category == FleetCarrierItemType.COMMODITIES_SELLING:
-                # Look up the display name because we don't have it in CAPI data
-                # Fall back to the internal name if we don't have a display name (probably out of date commodity.csv or rare_commodity.csv)
-                display_name: str = self.commodities.get(item[name_key], item[name_key])
-            else:
-                # Use the localised name from CAPI data
-                display_name: str = item[display_name_key]
-
-            if int(item[quantity_key]) > 0: result += f"{display_name} x {item[quantity_key]} @ {self._human_format_price(item['price'])}\n"
-
-        return result
+        try:
+            #Debug.logger.error(f"{category}")
+            items, name_key, display_name_key, quantity_key = self._get_items(category)
+            if items is None: return ""
+
+            result: str = ""
+
+            if category == FleetCarrierItemType.CARGO:
+                # Cargo is a special case because it can have multiple items with the same name so we have to sum them together
+                cargo = dict()
+                items = sorted(items, key=lambda x: x[display_name_key])
+                for item in items:
+                    if item[name_key] in cargo:
+                        cargo[item[display_name_key]] += int(item[quantity_key])
+                    else:
+                        cargo[item[display_name_key]] = int(item[quantity_key])
+                for key, value in cargo.items():
+                    result += f"{key} x {value}\n"
+                return result
+
+            if category == FleetCarrierItemType.LOCKER:
```

Comment:
Add a comment as to why LOCKER items are also a special case (you've noted why CARGO is a special case above).